pkgs/development/interpreters/supercollider/default.nix

```diff
@@ -16,19 +16,47 @@ mkDerivation rec {
     sha256 = "sha256-1QYorCgSwBK+SVAm4k7HZirr1j+znPmVicFmJdvO3g4=";
   };
 
+  patches = [
+    # add support for SC_DATA_DIR and SC_PLUGIN_DIR env vars to override compile-time values
+    ./supercollider-3.12.0-env-dirs.patch
+  ];
+
+  nativeBuildInputs = [ cmake pkg-config qttools ];
+
+  buildInputs = [
+    gcc libjack2 libsndfile fftw curl libXt qtbase qtwebengine qtwebsockets readline ]
+      ++ lib.optional (!stdenv.isDarwin) alsa-lib
+      ++ lib.optional useSCEL emacs;
```

Comment:
```suggestion
  buildInputs = [ gcc libjack2 libsndfile fftw curl libXt qtbase qtwebengine qtwebsockets readline ]
    ++ lib.optional (!stdenv.isDarwin) alsa-lib
    ++ lib.optional useSCEL emacs;
```

Reply:
I've force-pushed an update with this change. Thank you @SuperSandro2000!